Tools/createCellarFromRenvCache.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python3
```

Comment:
so I get this was perhaps easier to write, but this could also just be an R script that uses libarchive? jaspTools already uses an R interface to libarchive, for example.